dropwizard-logging/src/main/java/io/dropwizard/logging/AbstractAppenderFactory.java

```diff
@@ -263,17 +266,22 @@ public void setLayout(@Nullable DiscoverableLayoutFactory<E> layout) {
         }
     }
 
+    @SuppressWarnings("NullAway")
```

Comment:
I think it would be enough to add this suppression to line 279:
```
                 @SuppressWarnings("NullAway")
                 String logFormatWithTimeZone = logFormat.replace("%dwTimeZone", timeZone.getID());
```

Reply:
Yes, moved it. I was torn on how to fix this one. Thought the method annotation is more readable, even though a bit too broad.